transcoder/src/keyframes.go

```diff
@@ -135,6 +139,10 @@ func getKeyframes(path string, kf *Keyframe) error {
 		x := strings.Split(frame, ",")
 		pts, flags := x[0], x[1]
 
+		// true if there is no keyframes (e.g. in a file w/o video track)
+		if pts == "N/A" {
+			break
```

Comment:
Instead of breaking and making the normal path harder to read, I'd rather have a `return getDummyKeyframes(...)`

Reply:
Wrote a dedicated `getDummyKeyframes` that would call `GetInfo`. However, I found out that audio-only files with an embedded image DO have a single keyframe (at time 0). So kept this `break` as it is, and completed the condition after the loop. Let me know what you think

Reply:
good catch